csv/readcsv.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	runtime "github.com/ahsanfayaz52/EmployeeManagementSystem"
+	"github.com/ahsanfayaz52/EmployeeManagementSystem/models"
+)
+
+func main() {
+	rt, err := runtime.NewRuntime()
+	if err != nil {
+		panic(err)
+	}
+	root, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	filepath := "/Files/records.csv"
+	path := root + filepath
+	var rows = ReadCsv(path)
+	AddDataToDatabase(rows, rt)
+}
+
+// AddDataToDatabase adds employee data from csv to database.
+func AddDataToDatabase(rows [][]string, rt *runtime.Runtime) string {
+	for _, row := range rows {
+		var employee models.Employee
+		list := strings.Split(row[0], ",")
+
+		age, _ := strconv.Atoi(list[2])
+		salary, _ := strconv.Atoi(list[3])
+		phone, _ := strconv.Atoi(list[4])
+
+		employee.Name = list[0]
+		employee.Address = list[1]
+		employee.Age = age
+		employee.Salary = salary
+		employee.Phone = phone
+
+		_, err := rt.Service().AddEmployee(&employee)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+	return "data is added successfully"
+}
+
+// ReadCsv reads data from csv file and returns rows
+func ReadCsv(name string) [][]string {
+	f, err := os.Open(name)
+	if err != nil {
+		log.Fatalf("Cannot open '%s': %s\n", name, err.Error())
+	}
+	defer f.Close()
+	r := csv.NewReader(f)
+	r.Comma = ';'
```

Comment:
why do we need to do this?

Reply:
reader split each row when it finds semicolon , as rows are semicolon seperated